data/sql/updates/pending_db_world/rev_1677501542428607570.sql

```diff
@@ -0,0 +1,687 @@
+     (@HIGHLORD_MOGRAINE_ID, 0, 0, 1, 25, 0, 100, 513, 0, 0, 0, 0, 0, 53, 0, 16440, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Highlord Mograine Transform - On Reset - Start Waypoint (No Repeat)'),
+     (@HIGHLORD_MOGRAINE_ID, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 54, 5000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Highlord Mograine Transform - On Reset - Pause Waypoint (No Repeat)'),
+     (@HIGHLORD_MOGRAINE_ID, 0, 2, 3, 40, 0, 100, 0, 5, 0, 0, 0, 0, 103, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Highlord Mograine Transform - On Waypoint 5 Reached - Set Rooted On'),
+     (@HIGHLORD_MOGRAINE_ID, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 66, 0, 0, 0, 0, 0, 0, 19, 3976, 0, 0, 0, 0, 0, 0, 0, 'Highlord Mograine Transform - On Waypoint 5 Reached - Set Orientation Closest Creature \'Scarlet Commander Mograine\'');
+
+-- --
+-- Update Highlord Mograine's texts with broadcast IDs and move some of his
+-- emotes from code to data.
+
+SET @EMOTE_POINT := 25;
+SET @EMOTE_TALK := 1;
+
+SET @TEXT_RENAULT_GROUP_ID := 0;
+SET @TEXT_RENAULT_BROADCAST_ID := 12469;
+
+SET @TEXT_BETRAYAL_GROUP_ID := 1;
+SET @TEXT_BETRAYAL_BROADCAST_ID := 12471;
+
+SET @TEXT_FORGIVEN_GROUP_ID := 2;
+SET @TEXT_FORGIVEN_BROADCAST_ID := 12473;
+
+UPDATE `creature_text`
+    SET `BroadcastTextId` = @TEXT_RENAULT_BROADCAST_ID, `Emote` = @EMOTE_POINT
+    WHERE `GroupID` = @TEXT_RENAULT_GROUP_ID AND `ID` = 0 AND `CreatureID` = @HIGHLORD_MOGRAINE_ID;
+
+UPDATE `creature_text`
+    SET `BroadcastTextId` = @TEXT_BETRAYAL_BROADCAST_ID, `Emote` = @EMOTE_TALK
+    WHERE `GroupID` = @TEXT_BETRAYAL_GROUP_ID AND `ID` = 0 AND `CreatureID` = @HIGHLORD_MOGRAINE_ID;
+
+UPDATE `creature_text`
+    SET `BroadcastTextId` = @TEXT_FORGIVEN_BROADCAST_ID
+    WHERE `GroupID` = @TEXT_FORGIVEN_GROUP_ID AND `ID` = 0 AND `CreatureID` = @HIGHLORD_MOGRAINE_ID;
+
+-- --
+-- Move Commander Mograine's emote to creature_text and play the correct
+-- emote (he used to kneel).
+
+SET @EMOTE_BEG := 20;
+
+SET @FORGIVE_ME_GROUP_ID := 5;
+SET @FORGIVE_ME_BROADCAST_ID := 12472;
+
+UPDATE `creature_text`
+    SET `BroadcastTextId` = @FORGIVE_ME_BROADCAST_ID, `Emote` = @EMOTE_BEG
+    WHERE `GroupID` = @FORGIVE_ME_GROUP_ID AND `ID` = 0 AND `CreatureID` = @MOGRAINE_ID;
+
+SET @EMOTE_QUESTION := 6;
+
+SET @BUT_HOW_GROUP_ID := 4;
+SET @BUT_HOW_BROADCAST_ID := 12470;
+
+UPDATE `creature_text`
+    SET `BroadcastTextId` = @BUT_HOW_BROADCAST_ID, `Emote` = @EMOTE_QUESTION
+    WHERE `GroupID` = @BUT_HOW_GROUP_ID AND `ID` = 0 AND `CreatureID` = @MOGRAINE_ID;
+
+-- --
+-- Use sniffed data for Fairbanks's gossip texts
+
+-- 1
+SET @CURSE_LIFTED_BROADCAST_ID := 12480;
+SET @CURSE_LIFTED_TEXT := 'At last, the curse is lifted. Thank you, hero.';
+
+UPDATE acore_world.npc_text
```

Comment:
Remove `acore_world.` everywhere

Reply:
Done. What's the convention on resolving conversations here by the way? Should I mark this resolved, or wait for you to check it n you mark it resolved?

Reply:
if it's resolved it is resolved :)